trunk/SUAVE/Components/Energy/Networks/Battery_Propeller.py

```diff
@@ -65,6 +65,10 @@ def __defaults__(self):
         self.number_of_engines         = None
         self.voltage                   = None
         self.thrust_angle              = 0.0
+        self.pitch_command             = 0.0
+        self.nacelle_start             = 0.2
```

Comment:
I think if you're interested in more developed nacelle geometry we should start to think about them as an object. These parameters are totally unclear to me.

Reply:
nacelle geometry removed